tests/gcp/installer/util/test_resource/target_forseti_conf_server.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is an input file that is used by the Forseti prerequisites setup
+# wizard to generate an actual forseti_conf.yaml. It's better to work off
+# the forseti_conf.yaml.sample than this one if you are editing the config
+# by hand.
+
+global:
+    # gsuite
+    groups_service_account_key_file: /home/ubuntu/gsuite_key.json
+    domain_super_admin_email: test@forseti_test_gmail.com
+
+    # notification
+    # Remove or comment out the email properties if you do not want Forseti to
+    # send email notifications.
+    email_recipient: rec@forseti_test_gmail.com
+    email_sender: send@forseti_test_gmail.com
+    sendgrid_api_key: TESTING_API_KEY
+
+    # quota
+    # The pre-populated values are the defaults from GCP.
+    max_admin_api_calls_per_100_seconds: 8888
+    max_appengine_api_calls_per_second: 88
+
+##############################################################################
+
+scanner:
+
+    # Valid values are: debug, info, warning, error
+    loglevel: info
+
+    # Output path (do not include filename).
+    # If GCS location, the format of the path should be:
+    # gs://bucket-name/path/for/output
+    output_path: gs://OLD_TESTING_BUCKET/scanner_violations
+
+    # Rules path (do not include filename).
+    # If GCS location, the format of the path should be:
+    # gs://bucket-name/path/for/rules_path
+    # if no rules_path is specified, rules are
+    # searched in /path/to/forseti_security/rules/
+    rules_path: /home/ubuntu/forseti-security/rules
+
+    # Enable the scanners as default to true when integrated for Forseti 2.0.
+
+    scanners:
+        - name: bigquery
+          enabled: false
+        - name: bucket_acl
+          enabled: false
+        - name: cloudsql_acl
+          enabled: false
```

Comment:
More scanners here?  Should this be a full example?  If you want to test something truncated, perhaps that is a different test case?

Reply:
Added more scanners - replicated the yaml file from 1.0